frontends/web/src/components/settingsButton/settingsItem.tsx

```diff
@@ -1,26 +1,22 @@
-import { Component } from 'react';
+import { ReactNode } from 'react';
 import style from './settingsButton.module.css';
 
-interface SettingsItemProps {
-    optionalText?: string | null;
-    optionalIcon?: JSX.Element;
+type TProps = {
+  optionalText?: string | null;
```

Comment:
since you are here: I think the null should not be needed, can you remove it or does it explode somewhere?

Reply:
seems to be ok..